map/Makefile.docker

```diff
@@ -1,5 +1,5 @@
-.PHONY: serve
-serve:
+.PHONY: run
+run:
```

Comment:
On second thoughts, I'd like the `run` task here to run the static version of the site as it'll be much quicker to run.  But I can do that as a separate PR.

Reply:
cool!